filesystem/src/test/java/org/frankframework/filesystem/FileSystemSenderTest.java

```diff
@@ -452,24 +454,32 @@ public void fileSystemSenderListActionTestInFolder() throws Exception {
 		fileSystemSenderListActionTest("folder",2);
 	}
 
-	public void fileSystemSenderCreateFile(String folder, boolean fileAlreadyExists, boolean setCreateFolderAttribute) throws Exception {
+	public SenderResult fileSystemSenderCreateFile(String folder, boolean fileAlreadyExists, boolean setCreateFolderAttribute) throws Exception {
 		String filename = "create" + FILE1;
 
 		if(_folderExists(folder)) {
 			_deleteFolder(folder);
 		}
 		waitForActionToFinish();
 
+		if (fileAlreadyExists) {
+			createFile(folder, filename, "dummy-contents\n");
```

Comment:
Dit zou de helper moeten doen toch?
```suggestion
			_createFile(folder, filename, "dummy-contents\n");
```

Reply:
De method `createFile()`, zonder underscore, bestond al. Met underscore heeft geen parameter voor de content. Ik kan beide methods een naam met underscore geven.